templates/detail.html

```diff
@@ -10,13 +10,46 @@
         <!-- jQuery is already downloaded and installed for you -->
         <!-- In order to use jQuery in your JavaScript, this file needs to be added BEFORE your custom JavaScript -->
         <script type="text/javascript" src="/static/js/jquery.min.js"></script>
+
+
     </head>
     <body>
         <!-- Your HTML should be here -->
+        <a href="/"> <h3>Return to main page</h3></a>
+        <div class="container">
+            <div class="name"></div>
+            <div class="img"></div>
+            <h3>Types</h3>
+            <ul>
+                <div class="types"></div>
```

Comment:
The div should go outside the list instead of inside - all items within a `<ul>` tag at the top level should be `<li>`s

Reply:
like this:

`<ul class="types>
 (put your list items here)
</ul>
`